js/scripts.js

```diff
@@ -0,0 +1,12 @@
+
+var buttonElem = document.getElementById('addElem');
+var list = document.getElementById('list');
+
+
+buttonElem.addEventListener('click', function() {
+	var newListItem = document.createElement('li');
+	var getListItem = document.getElementsByTagName('li');
+	var numberListItem = getListItem.length;
```

Comment:
`getListItem` to tablica zawierająca wszystkie `li`

Za każdym dodanym elementem, dodajesz 2 nowe `li`, więc każdy kolejny będzie miał liczbę o 2 większą. Pierwsze `<li></li>` dodajrsz w #7 linijce, drugie w #10 - te z #10 są zbędne.